src/main/kotlin/net/maizegenetics/phgv2/cli/BuildMafVcf.kt

```diff
@@ -1,16 +1,48 @@
 package net.maizegenetics.phgv2.cli
 
 import com.github.ajalt.clikt.core.CliktCommand
-import com.github.ajalt.clikt.parameters.arguments.argument
-import com.github.ajalt.clikt.parameters.arguments.default
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.options.validate
 
 class BuildMafVcf : CliktCommand() {
 
+    //--maf-dir /my/maf/files -o /path/to/vcfs
```

Comment:
Do you want to keep this?

Reply:
Nope I meant to remove.  I will check the other ones.